From 79abc42e44a11cfb15487122e5b0bb12c65bedfb Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Wed, 4 Mar 2015 10:27:59 +0200
Subject: [PATCH 04/66] aarch64: defconfig: Rename the terminal name to ttyS0

	Change tty name at boot time to ttyS0

Change-Id: I0e7a46748b4bc787e68fb42a4a40cb6f45b514c8
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/17224
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
---
 configs/mvebu_armv8_le_defconfig      | 2 +-
 configs/mvebu_armv8_tiny_le_defconfig | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configs/mvebu_armv8_le_defconfig b/configs/mvebu_armv8_le_defconfig
index d6a5a0a..ac51958 100644
--- a/configs/mvebu_armv8_le_defconfig
+++ b/configs/mvebu_armv8_le_defconfig
@@ -1,6 +1,6 @@
 BR2_aarch64=y
 BR2_TOOLCHAIN_EXTERNAL=y
-BR2_TARGET_GENERIC_GETTY_PORT="ttyAMA0"
+BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_PACKAGE_BZIP2=y
 BR2_PACKAGE_DHRYSTONE=y
 BR2_PACKAGE_LMBENCH=y
diff --git a/configs/mvebu_armv8_tiny_le_defconfig b/configs/mvebu_armv8_tiny_le_defconfig
index 53d8b6f..be9ef1a 100644
--- a/configs/mvebu_armv8_tiny_le_defconfig
+++ b/configs/mvebu_armv8_tiny_le_defconfig
@@ -1,6 +1,6 @@
 BR2_aarch64=y
 BR2_TOOLCHAIN_EXTERNAL=y
-BR2_TARGET_GENERIC_GETTY_PORT="ttyAMA0"
+BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_TARGET_ROOTFS_CPIO=y
 BR2_TARGET_ROOTFS_TAR_GZIP=y
 BR2_PACKAGE_HOST_MKE2IMG=y
-- 
1.9.1

